src/contributors.njk

```diff
@@ -0,0 +1,15 @@
+---json
+{
+  "layout": "layout"
+}
+---
+
+<h1>Contributors</h1>
+
+<div class="contributors">
+{% for contributor in github.contributors %}
+  <a href={{ contributor.url }}>
```

Comment:
`contributor.url` unfortunately points to the API endpoint for the user rather than their profile URL. You can use `contributor.html_url`